admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -211,6 +227,25 @@ const ApiKeysPage = () => {
 									fullWidth
 								/>
 							</TableCell>
+							<TableCell>
+								<Checkbox
+									checked={txnMode === 0? false : true}
+									onChange={handleChange}
+									inputProps={{ 'aria-label': 'controlled' }}
```

Comment:
Was this copied from the docs?

Reply:
yes

Reply:
Okay i thought so, please either correct it or remove it please